src/resource.h

```diff
@@ -1,208 +1,209 @@
-//{{NO_DEPENDENCIES}}
```

Comment:
This whole file is marked as changed. Please edit in text editor so that only real changes show up in the diff.